README.md

```diff
@@ -93,6 +93,8 @@ This terraform module will add an IAM policy to the k8s cluster nodes roles to a
 * [`extra_grafana_dashboards`]: String(optional): Extra Grafana dashboards. From is the map structure of HELM.
 * [`extra_alertmanager_routes`]: String(optional): Extra alertmanager routes. Yaml format. (default: "")
 * [`extra_alertmanager_receivers`]: String(optional): Extra alertmanager receivers. Yaml format. (default: "")
+* [`k8s_admins_groups`]: List(optional): Groups that will be granted admin access to the k8s cluster. When using Dex and Kubesignin these will be GitHub teams in the form '<gh_org>:<gh_team>', for example 'skyscrapers:k8s-admins'. (default: [])
```

Comment:
Shouldn't we by default add our skyscrapers group?

Reply:
btw by default I also mean append the skyscrapers group as well when somebody adds groups of customers.

Reply:
Well, this is an open source module, so I don't think it should have any skyscrapers-specific stuff.

Reply:
This is an opinionated module already customised for us and we already add skyscrapers specific stuff https://github.com/skyscrapers/terraform-kubernetes/blob/master/base/main.tf#L138

Reply:
I agree it's an opinionated module, but that has nothing to do with locking it down to ourselves. Doing what you propose will mean that nobody else will be able to use this module as is, then we better make it private. I don't think that is worth it, for just setting a default value.

Another reason for not locking it down with our own groups and stuff: if a customer goes away, they'll need to remove some of our stuff, specially our access to the cluster, which is actually controlled by the variable we're talking about.

Reply:
@iuriaranda proposal we put it in here at this moment and once we get it automated we can remove it again.

Reply:
I still think it's not worth it, but ok...